platform/engineering/collab-cycle/architecture-intent-template.md

```diff
@@ -0,0 +1,79 @@
+# Architecture Intent Template
+
+__*** DRAFT *** DRAFT *** DRAFT ***__
+
+The Architecture Intent meeting helps your team build a solution that meets Veteran-facing Service Platform (VSP) engineering standards and lowers the potential for launch-blocking issues later in the development cycle.
+
+## What is the purpose of Architecture Intent meeting?
+
+The Architecture Intent meeting is less a formal presentation and more of a discussion.  It not only provides OCTO-DE and Platform with an early understanding of the product/feature your team wants to build, but is also an opportunity to collaborate and provide feedback on the intended implementation and surface any adjustments needed to meet VSP engineering standards. The focus is on making sure your code meets user needs within the constraints of the platform your building on.
+
+## When to schedule an Architecture Intent meeting?
```

Comment:
I think it is really good to have such a process in place, but how are final decisions made on adoption of the new thing?  

Let's say my team wants to innovate at the VA and move to a new technology that would inarguably result in better code quality and therefore improve the veteran experience.  Let's say this new technology is called Typescript.  

When I bring this new idea to whomever is reviewing the change, are they familiar with the technology?  A lot people are afraid to admit that they just haven't used or aren't familiar with the new thing you are trying to do.  It therefore becomes very difficult to get buy-in from a group.  What happens is that a single person raising a concern out of ten meetings you have over six  months is enough to shoot down the idea.   

The idea of a single place where new ideas and technologies can be vetted and centralized is great, but I really worry that it is yet another roadblock in what is already a very difficult place to innovate.